vllm/engine/arg_utils.py

```diff
@@ -1041,6 +1043,10 @@ def add_cli_args(parser: FlexibleArgumentParser) -> FlexibleArgumentParser:
             "--structured-outputs-config", **vllm_kwargs["structured_outputs_config"]
         )
 
+        vllm_group.add_argument(
+            "--optimization-level", **vllm_kwargs["optimization_level"]
+        )
+
```

Comment:
If we are adding this, would it be appropriate to move `-O` from `--compilation-config` to here?

Reply:
We should map `-On` on the command line to opt level but I think `-O.<...>` should remain mapped to config properties. Until we have a better solution (`-C.`?) we shouldn't change it as it'll break backwards compatibility.

Reply:
Yeah thanks for raising this, very easy to get used to the status quo even when it's not intuitive 😅 we'll merge this the way it is currently and I'll write an RFC for changing `-O.<key>` to `-cc.<key>`